### ISSUE TITLE
Theme Options > Layout > Other Homepage Display Options term dropdown menu has "List Below" but this does nothing

### ISSUE THREAD

**Issue by jackbrighton** [INN/Largo#1205]
In Largo 0.5.4, Theme Options > Layout > Other Homepage Display Options has a dropdown menu to select where Categories and Tags are shown with each post on the homepage. Options are:

- Single Tag Above
- List Below
- None

The List Below option is supposed to display a list of tags below the story's excerpt, but no tags display when this option is selected. This is true on all six of the Home Templates.

**Comment by aschweigert** [comment]
This option should probably just be removed (or at least moved), we haven't shown tags on the homepage for some time (see: https://github.com/INN/Largo/blob/master/partials/content.php#L56) because sites tend to overuse tags and it looks really junky. They are still used on internal pages so maybe we should just move the option to a more sensible spot in the theme options.

**Comment by jackbrighton** [comment]
We still show the Single Tag Above though, which can come in handy for accessing the tag archive page for any given post.

**Comment by aschweigert** [comment]
Right, which I think should be the default behavior because choosing a top term is an editorial decision and should be more informative to readers than just showing a full list.

**Comment by jackbrighton** [comment]
Agreed!
